build.gradle

```diff
@@ -32,24 +31,29 @@ dependencies {
     implementation(group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1')
     implementation(group: 'javax.el', name:'javax.el-api', version:'3.0.0')
     // Groovy
-    implementation(group:'org.codehaus.groovy', name:'groovy-all', version:'3.0.8')
+    implementation (group: 'org.apache.groovy', name: 'groovy-all', version: '4.0.15', ext: 'pom')
 
     // Spring
     implementation(group:'org.springframework.boot', name:'spring-boot-starter-data-jpa', version:"${springBootVer}")
     implementation(group:'org.springframework.boot', name:'spring-boot-starter-security', version:"${springBootVer}")
-    implementation(group:'net.rakugakibox.spring.boot', name:'logback-access-spring-boot-starter', version:'2.8.0')
+    implementation(group:'net.rakugakibox.spring.boot', name:'logback-access-spring-boot-starter', version:'2.11.0')
 
     implementation(group:'com.authy', name:'authy-java', version:'1.5.1')
-    implementation(group:'com.twilio.sdk', name:'twilio', version:'8.11.0')
-    implementation(group:'org.xerial', name:'sqlite-jdbc', version:'3.34.0')
+    implementation(group:'com.twilio.sdk', name:'twilio', version:'8.31.1')
+    implementation(group:'org.xerial', name:'sqlite-jdbc', version:'3.41.2.2')
 
     testImplementation(group: 'junit', name:'junit', version:'4.13.2')
     testImplementation(group:'org.springframework.boot', name:'spring-boot-starter-test', version:"${springBootVer}")
-    
+
+    // Constraints
+    constraints {
+        implementation('org.json:json:20231013') because("transitive dependecy of authy-java has vulnerabilites issues")
```

Comment:
I think using a closure is a bit more readable here, but it's fine this way as well 🙂 
```suggestion
        implementation('org.json:json:20231013') {
          because("transitive dependency of authy-java has vulnerabilities issues")
        }
```

**PS:** There's a couple of typos on `because(..)` message as well 😅